extensions/ql-vscode/src/pure/helpers-pure.ts

```diff
@@ -55,3 +55,11 @@ export function getErrorStack(e: any) {
 export function asError(e: any): Error {
   return e instanceof Error ? e : new Error(String(e));
 }
+
+/**
+ * Pluralizes a word.
+ * Example: Returns "N repository" if N is one, "N repositories" otherwise.
+ */
+export function pluralize(numItems: number | undefined, singular: string, plural: string): string {
```

Comment:
`helpers-pure` is a generic bucket that I'd personally like to see us gradually move away from. How about we move this function to a new file/module e.g. `word`?

Reply:
Good point! Moved in https://github.com/github/vscode-codeql/pull/1639/commits/c85338d11a433eb0a69ad4677e66cef54f847f2d 🦞